### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by deekshanee** [langflow-ai/langflow#3281] [General]
Hi team

I contributed to the langflow code , when my name will be under contributor list ?
https://github.com/langflow-ai/langflow/pull/3279

**Comment by deekshanee** [comment]
@lucaseduoli  can you please help here?

**Comment by severfire** [comment]
I wonder if there could be made distinction between testers, and coders :-) some of us are not coders, yet they know how to test and will be happy to do so - and testers are very important as well. also, after all we are all one big family :-)

**Reply by deekshanee** [reply]
Both are equally important , As a full stack developer I can understand importance of testing as well, I fixed the code and also I have many other things to fix in future :)
